marsha/settings.py

```diff
@@ -0,0 +1,99 @@
+"""Django settings for marsha project.
+
+Uses django-environ to load some config from the environment
+
+"""
+
+
+import environ
```

Comment:
Do you know `django-configurations`[1]? I particularly like the class-base approach to define settings. And they also support 12 factors app derived settings.

[1] https://github.com/jazzband/django-configurations

Reply:
As I said in my answer to sam:

> If it's your choice I'll follow you. Not fan of this kind of heavy machinery that totally changes how settings are managed. But it's only my opinion.

Reply:
ok sam decided to go with this heavy machinery ;)